lib/Dialect/Torch/Transforms/DecomposeComplexOps.cpp

```diff
@@ -9127,6 +9147,7 @@ class DecomposeComplexOpsPass
     addPatternIfTargetOpIsIllegal<DecomposeAtenWhereScalarSelfOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenNanToNumOp>(patterns);
     addPatternIfTargetOpIsIllegal<DecomposeAtenMaskedFillScalarOp>(patterns);
+    addPatternIfTargetOpIsIllegal<DecomposeAtenMaskedFillTensorOp>(patterns);
```

Comment:
also add `target.addIllegalOp<AtenMaskedFillTensorOp>()` in `lib/Dialect/Torch/Transforms/LowerToBackendContract.cpp`